packages/alfa-web/src/page.ts

```diff
@@ -64,11 +64,11 @@ export class Page
     return this._device;
   }
 
-  public toJSON(): Page.JSON {
+  public toJSON(options?: Node.SerializationOptions): Page.JSON {
     return {
       request: this._request.toJSON(),
       response: this._response.toJSON(),
```

Comment:
Since we're touching this file for this reason, we should as well future-proof it.
```suggestion
      request: this._request.toJSON(options),
      response: this._response.toJSON(options),
```